config/passport.js

```diff
@@ -3,26 +3,15 @@ var LocalStrategy    = require('passport-local').Strategy;
 var FacebookStrategy = require('passport-facebook').Strategy;
 var GoogleStrategy   = require('passport-google-oauth').OAuth2Strategy;
 
-// load up the user model
-var configDB = require('./database.js');
```

Comment:
I moved all this into data.js. Basically, if you want to look-up data, require data.js, and use the objects/methods it exports. This way, the model only gets loaded once.